recipes/GillesPy2/meta.yaml

```
@@ -0,0 +1,57 @@
+
+{% set name = "gillespy2" %}
+{% set version = "1.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8e0eff83a8e29e0fe0c72252505fe216cd8cf95feb3d3714a9309d8c81e72656
+  
+build:
+  number: 0
+  skip: True  # [py<35]
```

This looks like it can be noarch python? If (and only if) that is the case you can keep the python keep the Python versions constraint by making the dependencies both `python >=3.5` instead of `python`:
```suggestion
  noarch: python
```